install/installer/pkg/config/v1/config.go

```diff
@@ -121,6 +121,7 @@ type Config struct {
 
 	Database Database `json:"database" validate:"required"`
 
+	// Deprecated.
 	MessageBus *MessageBus `json:"messageBus,omitempty"`
```

Comment:
This ensures that existing configs which do mention the messagebus do continue to work. This allows us to remove it from other places (dedicated) before removing it from code.